e2e/src/app.e2e-spec.ts

```diff
@@ -1,14 +1,57 @@
 import { AppPage } from './app.po';
-
+import { browser, by, element, $ } from 'protractor';
 describe('new App', () => {
   let page: AppPage;
 
   beforeEach(() => {
     page = new AppPage();
   });
 
-  it('should display welcome message', () => {
-    page.navigateTo();
-    expect(page.getPageTitle()).toContain('Tab One');
+  it('Has MapConU as the Title ', () => {
+    browser.get('/');
+    let x = element(by.id('campus-change')).getText();
+    expect(browser.getTitle()).toContain('MapConU');
+  });
+  it('Should toggle Campuses when selected at top of the UI', () => {
+    browser.get('/');
+    //set location to sgw
+    element(by.xpath("/html/body/app-root/ion-app/header/app-outdoor-navigation-toolbar/ion-header/ion-toolbar/ion-item/ion-select")).click();
+    browser.driver.sleep(500);
+    element(by.xpath("/html/body/app-root/ion-app/ion-popover/div/div[2]/ion-select-popover/ion-list/ion-radio-group/ion-item[1]")).click();
+
+    //check location is set to SGW
+    expect(element(by.id("campus-change")).getText()).toContain("SGW Campus");
+    expect(element(by.css('ng-reflect-longitude="-73.578041"'))).toBeDefined();
+    expect(element(by.css('ng-reflect-latitude="45.495729"'))).toBeDefined();
+
+    //set location to Loyola
+    element(by.xpath("/html/body/app-root/ion-app/header/app-outdoor-navigation-toolbar/ion-header/ion-toolbar/ion-item/ion-select")).click();
+    browser.driver.sleep(500);
+    element(by.xpath("/html/body/app-root/ion-app/ion-popover/div/div[2]/ion-select-popover/ion-list/ion-radio-group/ion-item[2]")).click();
+
+    //check locations is set to loyla
+    expect(element(by.id("campus-change")).getText()).toContain("Loyola Campus");
+    expect(element(by.css('ng-reflect-longitude="-73.640452"'))).toBeDefined();
+    expect(element(by.css('ng-reflect-latitude="45.45824"'))).toBeDefined();
+
   });
+  it("Shows Current Position Marker", () => {
+    browser.get('/');
+    let ele = element(by.xpath("/html/body/app-root/ion-app/app-google-map/div/agm-map/div[1]/div/div/div[1]/div[3]/div/div[3]")).isPresent();
+    expect(ele).toBeTruthy();
+  });
```

Comment:
Should commented out code be left out?

Reply:
I believe that in the short term that it would cleaner, but in the next sprint the test should be implemented. Therefore I believe it should be left in so it will not have to be rewritten from scratch.

Reply:
gotcha!